modules/ROOT/pages/auto-updates.adoc

```diff
@@ -54,10 +56,12 @@ storage:
       contents:
         inline: |
           [updates]
-          strategy = "fleet_lock"
-          [updates.fleet_lock]
-          base_url = "https://example.com/lock-manager/"
+          strategy = "periodic"
+          [[updates.periodic.window]]
```

Comment:
I need to learn more TOML. The `[[ ]]`` syntax is new to me.

Reply:
That's an entry in an [array of tables](https://github.com/toml-lang/toml/blob/master/toml.md#array-of-tables).